test/rcall_test.jl

```diff
@@ -0,0 +1,29 @@
+using RCall
```

Comment:
I'd rather not have our tests depend on R. Hardcoding test values known to be the same as R is enough.